swagger.yaml

```diff
@@ -3153,6 +3153,48 @@ paths:
             $ref: '#/definitions/error'
 
 definitions:
+  baremetal_platform:
+    description: Baremetal platform specific configuration upon which to perform the installation
+    type: object
+    properties:
+      provisioningNetwork:
+        type: string
+        enum: [Disabled, Enabled]
+
```

Comment:
Why is this needed?  We don't support setting this property.

Reply:
@avishayt Do we not want the API to support setting all platform properties?
And if not, completely remove the `baremetal_platform` structure?

Reply:
We want to expose only the properties that we want to support.  Yes, remove the `baremetal_platform` structure.